tests/integration/targets/filter_list/tasks/lists_mergeby_default.yml

```
@@ -0,0 +1,166 @@
+---
+
+- name: Debug ansible_version
+  debug:
+    var: ansible_version
+  when: debug_test|d(false)|bool
+  tags: t0
+
+- name: 1.Test lists merged by attribute name
```

```suggestion
- name: 1. Test lists merged by attribute name
```
Same for the others.